packages/zowe-explorer/README.md

```diff
@@ -83,7 +89,33 @@ Before using the base profile functionality, ensure that you have [Zowe CLI](htt
 
 For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
 
-### Sample use cases
+#### Log in to the Authentication Service
+
+Use the Log in to the **Authentication Service** feature to regenerate a new token for your base profile.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Right-click your profile.
+3. Select the **Log in to Authentication Service** option.
+
+   You will be prompted to enter your username and password beforehand.
+
+The token is stored in the default base profile .yaml file.
+
+**Note:** There is a known issue that tokens issued in Zowe Explorer, from an API Mediation Layer instance accepting self-signed certificates, are rejected with `Error: self signed certificate in certificate chain`, even if the reject unauthorized flag is set to false in the base profile. You can continue to use tokens generated from Zowe CLI in Zowe Explorer.
+
+If you do not want to store your token, request from the server to end the session of your token. Use the **Log out from Authentication Service** feature to invalidate the token.
```

Comment:
If you do not want to store your token, you can request the server to end your session token. 

Does that sound better, or not?